### PR TITLE
News Card Component

### PR DESCRIPTION
**Pull request by ModulesUnraveled**
This PR creates the News Card Component

### CODE REVIEW

File: 02-molecules/cards/NewsCard/NewsCard.component.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      <div className={bem(block, 'content')}>
+        <div className={bem(block, 'content-top')}>
+          {/* Heading */}
+          {heading && (
+            <Heading
+              level="2"
+              children={heading}
+              block={block}
+              element="heading"
+            />
```
Comment (nJim) [inline 02-molecules/cards/NewsCard/NewsCard.component.js:40]:
```
<Heading
  level="2"
  block={block}
  element="heading"
>
  { heading }
</Heading>
```

File: 02-molecules/cards/NewsCard/NewsCard.component.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+          {image && <Image outputImage={outputImage} image={image} />}
+        </div>
+        {/* Details */}
+        <div className={bem(block, 'meta--details')}>
+          <div className={bem(block, 'date')}>
+            <span className={bem(block, 'day')}>{day}</span>
+            <span className={bem(block, 'month')}>{month}</span>
+          </div>
+          {/* Tag */}
+          {tag && <Paragraph children={tag} block={block} element="tag" />}
```
Comment (nJim) [inline 02-molecules/cards/NewsCard/NewsCard.component.js:71]: `{tag && <Paragraph block={block} element="tag">{ tag }</Paragraph>}`

File: 02-molecules/cards/NewsCard/NewsCard.component.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+            <Heading
+              level="3"
+              children={subheading}
+              block={block}
+              element="subheading"
+            />
+          )}
+        </div>
+        {/* Body */}
+        {body && <Paragraph children={body} block={block} element="body" />}
```
Comment (nJim) [inline 02-molecules/cards/NewsCard/NewsCard.component.js:53]: `{body && <Paragraph block={block} element="body">{ body }</Paragraph>}`

File: 02-molecules/cards/NewsCard/NewsCard.component.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+            <Heading
+              level="2"
+              children={heading}
+              block={block}
+              element="heading"
+            />
+          )}
+          {/* Subheading */}
+          {subheading && (
+            <Heading
```
Comment (nJim) [inline 02-molecules/cards/NewsCard/NewsCard.component.js:44]:
```
<Heading
  level="3"
  block={block}
  element="subheading"
>
  { subheading }
</Heading>
```

File: 02-molecules/cards/cards.stories.js [hunk 5]
```diff
@@ -43,3 +44,17 @@ export const cardWithBackground = () => (
     nibh pulvinar a.
   </Card>
 );
+export const newsCard = () => (
+  <NewsCard
+    image={{
+      alt: 'Random Image from Picsum',
+      src: 'https://picsum.photos/400/300.jpg',
```
Comment (nJim) [inline 02-molecules/cards/cards.stories.js:51]: Want to match the Twig demo site? It uses src "https://picsum.photos/1280/720".

File: 02-molecules/cards/NewsCard/NewsCard.component.js [hunk 6]
```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Image from '../../../01-atoms/images/ResponsiveImage/ResponsiveImage.component';
+import { imageTypeProps } from '../../../01-atoms/images/Image/Image.component';
+import bem from '../../../_utils/bem';
+import Heading from '../../../01-atoms/text/Heading.component';
+import Paragraph from '../../../01-atoms/text/Paragraph.component';
+import Button from '../../../01-atoms/buttons/Button.component';
```
Comment (nJim) [inline 02-molecules/cards/NewsCard/NewsCard.component.js:8]: The news card component on the Twig site uses a Link not a button.

File: 02-molecules/cards/cards.stories.js [hunk 7]
```diff
@@ -43,3 +44,17 @@ export const cardWithBackground = () => (
...
+    }}
+    heading="This is the card heading"
+    subheading="Curabitur non nulla sit amet nisl tempus convallis quis ac lectus"
+    body="Ellentesque in ipsum id orci porta dapibus. Mauris blandit aliquet elit, eget tincidunt nibh pulvinar a. Mauris blandit aliquet elit, eget tincidunt nibh pulvinar a."
+    link_text="Read more about this card"
+    date="2119-12-09T12:00:00"
+    tag="Fun Stuff"
+  />
```
Comment (nJim) [inline 02-molecules/cards/cards.stories.js:59]: Please add a button to the story.

Should this use the same test copy from the Twig Site? https://dev-westernuni.pantheonsite.io/storybook/?path=/story/molecules-cards--news-card

File: 02-molecules/cards/cards.stories.js (hunk 5 above)
Comment (ModulesUnraveled) [reply 02-molecules/cards/cards.stories.js:51]: yep

File: 02-molecules/cards/cards.stories.js (hunk 7 above)
Comment (ModulesUnraveled) [reply 02-molecules/cards/cards.stories.js:59]: Sure

**Comment by ModulesUnraveled** [comment]
I think I've addressed all comments.

**Pull request merged**